src/show/pages/public/Constructor/index.jsx

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+const Constructor = () => {
+  const { id } = useParams();
+  const [getConstructorData, setGetConstructorData] = useState({});
```

Comment:
Not a good name for those.

Reply:
It ok to call them just constructorData and setConstructorData?